packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/quanticGeneratedAnswer.e2e.ts

```diff
@@ -8,6 +8,8 @@ const fixtures = {
   insight: testInsight,
 };
 
+const exampleAnswerConfigurationId = '3bb64276-0d26-4afc-be08-b0a8f2295de4';
```

Comment:
this is tied to a "real" id right? Is the ID name quite clear in the coveo org we use for these tests? Just to make sure its easy to find if we need to debug/refer to it later on.

Reply:
Yes, but we are not really benefiting from the real answer configuration as we are mocking the streaming, so even having a fake id here would do.